istio.io/configmap-nginx.yaml

```diff
@@ -112,6 +113,11 @@ data:
         # the github side are to the vanity domain.  DNS points the vanity
         # domain at this nginx.
         location / {
+          # Redirect all version prefixed URLs for non-documentation to
+          # the latest and greatest version of the site, e.g. faq,
+          # blogs.
+          rewrite ^/v-[0-9]+\.[0-9]+(/?.*)$ https://istio.io$1 last;
```

Comment:
I don't think the / is optional there - we don't want to rewrite v-0.1alpha/foo to istio.ioalpha/foo